src/main/java/com/likeservice/Service/LikeService.java

```diff
@@ -1,11 +1,100 @@
 package com.likeservice.Service;
 
+import com.likeservice.ConstantFile.ConstantFiles;
+import com.likeservice.Exception.LikesNotFoundException;
+import com.likeservice.Feign.UserService;
+import com.likeservice.Model.Like;
+import com.likeservice.Model.LikeDto;
 import com.likeservice.Repository.LikeRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import java.time.LocalDateTime;
+
 @Service
 public class LikeService {
     @Autowired
     private LikeRepository likeRepository;
+
+    @Autowired
+    private UserService userFeign;
+
+    public String deleteLikeID(String likeId) {
+        if (likeRepository.findById(likeId).isPresent()) {
+            likeRepository.deleteById(likeId);
+            return ConstantFiles.passCode;
```

Comment:
Why are you calling it as ConstantFiles ?

Reply:
Done